app/Http/routes.php

```diff
@@ -102,8 +102,9 @@
     Route::delete('signups/{signup}', 'Three\SignupsController@destroy');
 
     // posts
+    Route::post('posts', 'Three\PostsController@store');
```

Comment:
Waaaaaait, does this method exist? 👀

Reply:
omg that is so weird I don't know where it went!!

Reply:
oh wait wait ok that's right - it goes through the `PostRequests` trait so actually goes to the `store` method here: https://github.com/DoSomething/rogue/blob/master/app/Http/Controllers/Traits/PostRequests.php#L19.

Reply:
if this `PostRequests` trait is confusing people then we can kill it, I just thought it would be easier than having the same method in a bunch of different places.

Reply:
OH! Tricky, I totally missed that. I'm cool with keeping it for the v2 endpoints, but I think we should have a separate `store` method here for the v3 version of this route (since we'll want to customize some of the implementation details, like where we get `northstar_id` from).